kitsune/users/tests/test_api.py

```diff
@@ -187,6 +187,13 @@ class TestUserView(TestCase):
     def setUp(self):
         self.client = APIClient()
 
+    def test_usernames_with_periods(self):
+        p = ProfileFactory(user__username='something.something')
+        url = reverse('user-detail', args=[p.user.username])
```

Comment:
The `UserFactory` is better than the `user` model maker, in that it will create a profile for all users. Given that, you should be able to do this instead, which saves some typing:

```python
u = UserFactory(username='something.something')
url = reverse('user-detail', args=[u.username])
```